packages/devel/elfutils/package.mk

```diff
@@ -26,6 +26,7 @@ PKG_CONFIGURE_OPTS_HOST="utrace_cv_cc_biarch=false \
                          --disable-programs \
                          --disable-nls \
                          --disable-debuginfod \
+			 --disable-libdebuginfod \
```

Comment:
indentation

Reply:
Love them tabs

Reply:
we don't do tabs at all :) and v4l-utils are likely a mistake

```
$ grep -rP '\t' --include="package.mk" ./packages/
./packages/sysutils/v4l-utils/package.mk:	--enable-bpf \
./packages/sysutils/v4l-utils/package.mk:	--enable-static \
./packages/sysutils/v4l-utils/package.mk:	--disable-shared \
./packages/sysutils/v4l-utils/package.mk:	--disable-doxygen-doc"
./packages/sysutils/v4l-utils/package.mk:# *		rc-rc6-mce	rc6_mce.toml
./packages/sysutils/v4l-utils/package.mk:*		rc-rc6-mce	libreelec_multi.toml
./packages/sysutils/v4l-utils/package.mk:meson-ir	rc-empty	libreelec_multi.toml
./packages/tools/rpi-eeprom/package.mk:	_basedir="$(basename "${_dir}")"
```

Reply:
👍. Not a fan either.

Reply:
Fixed the 2 other package.mk files - https://github.com/LibreELEC/LibreELEC.tv/pull/4814 remaining 3 lines of tabs in the v4l-utils/package.mk are supposed to be there.